src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -645,6 +645,45 @@ def set_bucket_versioning(self, bucket_name, versioning_state, mfa_delete,
                                      http_status_code=error.status_code,
                                      component='rgw')
 
+    @RestClient.api_get('/{bucket_name}?encryption')
+    def get_bucket_encryptyion(self, bucket_name, request=None):
+
+        # pylint: disable=unused-argument
+        result = request()
+        if 'SSEAlgorithm' not in result:
+            result['SSEAlgorithm'] = 'No Encryption'
+        return result
+
+    @RestClient.api_put('/{bucket_name}?encryption')
+    def set_bucket_encryption(self, bucket_name, keyId, bucketKeyEnabled, sseAlgorithm, request=None):
```

Comment:
I understand that here the `mixedCase` convention has been used, but given there's already some `snake_case` over there, let's try and use the proper syntax for variable names:
```suggestion
    def set_bucket_encryption(self, bucket_name, keyId, bucketKeyEnabled, sse_algorithm, request=None):
```